plugin-flex-ts-template-v2/src/feature-library/custom-transfer-directory/README.md

```diff
@@ -0,0 +1,47 @@
+# custom-transfer-directory
+This feature enables the replacement of the queue transfer directory enabling the following behavior
+
+- queue
+  - render different transfer icons for chat channel
+  - enable the use of real time data to 
+      - present queue insights such as # of tasks or # agents available
+      - other data available such as queue wait time but further work required to show it on screen
+      - filter out queues that have no agents available
+  - provide an improved starting point for augmenting queue transfer list with custom data (imagine the need to filter queues based on skills required to transfer to those queues)
+  - provide the ability to enforce queue filters by worker
+  - provide ability to enforce global queue filter to filter out system queues.
+
+
+# flex-user-experience
+
+Example call transfer
+
+![alt text](screenshots/flex-user-experience-queue-transfer.gif)
+
+
+# setup and dependencies
+
+Enable the feature in the flex-config asset for your environment. 
+
+``` javascript
+"custom_transfer_directory": {
+  "enabled": true, // globally enable or disable the feature
+  "use_legacy_search_icon": true, // use legacy icon (recommended if mixing queue overrided with OOTB overrride)
+  "worker" : {
+    "enabled": false // enable the custom worker tab - not currently developed, still to come
+  },
+  "queue" : {
+    "enabled": true, // enable the custom queue tab
+    "show_only_queues_with_available_workers": true, 
+    "show_real_time_data" : true, // tool tup for queues will show real time data instead of queue name
+    "enforce_queue_filter_from_worker_object": true, // when true, if `worker.attributes.enforcedQueueFilter` is present, it will be enforced, otherwise ignored
```

Comment:
I wonder if this should instead not be enabled by default given this attribute isn't present by default.

Reply:
i think its better to have it enabled by default because the absence of the field on the worker is ignored and simply adding it to the worker will have it evaluated for use so its less overhead in configuring and managing.